libs/features/footer-promo.js

```diff
@@ -42,4 +44,6 @@ export default async function initFooterPromo(footerPromoTag, footerPromoType) {
   document.querySelector('main > div:last-of-type').insertAdjacentElement('afterend', section);
   await loadFragment(a);
   section.classList.add('section');
+  const sections = document.querySelectorAll('main > div');
```

Comment:
Could you reuse the selector from L44 or otherwise just use 
```suggestion
  const lastSection = document.querySelectorAll('main > div:last-of-type');
```

Reply:
@mokimo I am only selecting to get the number of sections. I use the length of the results in the function call on the next line.  I don't think the suggested update would still get me this info.